packages/neo-one-smart-contract-lib/src/token/SecondaryNEP5Token.ts

```diff
@@ -0,0 +1,168 @@
+// tslint:disable readonly-keyword readonly-array no-object-mutation strict-boolean-expressions
+import {
+  Address,
+  constant,
+  Contract,
+  createEventNotifier,
+  Fixed,
+  MapStorage,
+  SmartContract,
+} from '@neo-one/smart-contract';
+import { Secondary } from '../ownership';
+
+interface TokenPayableContract {
+  readonly approveReceiveTransfer: (from: Address, amount: Fixed<0>, asset: Address) => boolean;
+  readonly onRevokeSendTransfer: (from: Address, amount: Fixed<0>, asset: Address) => void;
+}
+
+export abstract class SecondaryNEP5Token extends Secondary(SmartContract) {
```

Comment:
It's not clear to me what adding `Secondary` here does. We don't appear to be using the mixin anywhere in this class.